doc/usecases/map_timeline.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Map Timeline View
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can view the map at different points in time by using the timeline feature
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and selected the map view.
+- **Main success scenario:**
+  The user selects the timeline view and uses the scroll bar with a month granularity or date field to navigate to a different point in time. 
```

Comment:
Is it possible that the scroll bar visually indicates in which months there are changes and at which months the map is empty (without plants)?

Reply:
Should be possible yes

Reply:
That would be amazing :heart: